internal/detectors/exploitation/command_execution.go

```
@@ -0,0 +1,127 @@
+	copy(regexes, commandEvidenceRegexes)
+
+	minEnv := minEnvAssignments
+
+	if cfg != nil {
+		if extra, ok := cfg["evidence_patterns"].([]string); ok {
+			for _, p := range extra {
+				re, err := regexp.Compile(p)
+				if err != nil {
+					return nil, err
+				}
+				regexes = append(regexes, re)
```

golangci-lint (2.12.2)

[error] append to slice `regexes` with non-zero initialized length

(makezero)